.github/workflows/docs.yaml

```diff
@@ -0,0 +1,31 @@
+name: Build Docs
+on:
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+jobs:
+  build_docs:
+    name: ${{ matrix.python_version }} build docs
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python_version: ["3.10"]
+    steps:
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python_version }}
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+      - name: Build gridstatus package
+        run: make package
+      - name: Install gridstatus
+        run: |
+          python -m pip install "unpacked/[docs]"
+      - name: Build docs
+        run: make -C docs/ -e "SPHINXOPTS=-j auto" clean html
```

Comment:
There is no `SPHINXOPTS="-W"` here which means that warnings will NOT be treated as errors.